src/main/java/com/github/osisoft/ocs_sample_library_preview/dataviews/DataViewClient.java

```diff
@@ -739,16 +837,56 @@ public ResponseWithLinks getDataViewStoredData(String namespaceId, String dataVi
      *                    default value.
      * @param count       The requested page size. The default value is 1000. The
      *                    maximum is 250,000.
+     * @param verbose     Whether the response from OCS should be verbose (containing
```

Comment:
Is this actually a param below?

Reply:
thanks, lots of copy/paste, lost track of this one